tensorflow/contrib/learn/python/learn/estimators/linear.py

```diff
@@ -148,22 +148,16 @@ def _linear_model_fn(features, labels, mode, params, config=None):
       parent_scope,
       values=tuple(six.itervalues(features)),
       partitioner=partitioner) as scope:
-    if joint_weights:
-      logits, _, _ = (
-          layers.joint_weighted_sum_from_feature_columns(
-              columns_to_tensors=features,
-              feature_columns=feature_columns,
-              num_outputs=head.logits_dimension,
-              weight_collections=[parent_scope],
-              scope=scope))
-    else:
-      logits, _, _ = (
-          layers.weighted_sum_from_feature_columns(
-              columns_to_tensors=features,
-              feature_columns=feature_columns,
-              num_outputs=head.logits_dimension,
-              weight_collections=[parent_scope],
-              scope=scope))
+    
+    layer_fn = layers.joint_weighted_sum_from_feature_columns if joint_weights \
```

Comment:
per the [google python style guide](https://google.github.io/styleguide/pyguide.html?showone=Conditional_Expressions#Conditional_Expressions) since this is longer than a 1 liner could you change it to a regular if statement?